core-java-modules/core-java-time-measurements/src/test/java/com/baeldung/time/InstantUnitTest.java

```diff
@@ -1,33 +1,28 @@
 package com.baeldung.time;
 
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.mockito.MockedStatic;
 
 import java.time.Clock;
 import java.time.Instant;
 import java.time.ZoneId;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
-import static org.powermock.api.mockito.PowerMockito.when;
+import static org.mockito.Mockito.mockStatic;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({ Instant.class })
 public class InstantUnitTest {
 
     @Test
     public void givenInstantMock_whenNow_thenGetFixedInstant() {
         String instantExpected = "2014-12-22T10:15:30Z";
         Clock clock = Clock.fixed(Instant.parse(instantExpected), ZoneId.of("UTC"));
         Instant instant = Instant.now(clock);
-        mockStatic(Instant.class);
-        when(Instant.now()).thenReturn(instant);
 
-        Instant now = Instant.now();
-
-        assertThat(now.toString()).isEqualTo(instantExpected);
+        try (MockedStatic<Instant> mockedStatic = mockStatic(Instant.class)) {
```

Comment:
Have you checked that this piece of code isn't mentioned in the article?

Reply:
We will have to update this article:
https://www.baeldung.com/java-override-system-time

I think power mock doesn't support JDK 17 yet, so we will have to update it to use Mockito as it now has the capability to mock static methods as well.